<launch>
 <node pkg="motor_communication" type="motor_communication_node" name="motor_communication" output="screen">
    <param name="serial_port" value="/dev/ttyUSB2" />
    <param name="baud_rate" value="9600" />
  </node>
</launch>
